src/Uno.Extensions.Authentication.Msal/Uno.Extensions.Authentication.Msal.UI.csproj

```diff
@@ -18,6 +18,19 @@
 			<Pack>true</Pack>
 		</Content>
 	</ItemGroup>
+
+	<PropertyGroup Condition="$(TargetFramework.ToLower().StartsWith('net6')) and $(_IsIOS)">
+		<DefineConstants>$(DefineConstants);IOS1_0;XAMARIN;XAMARIN_IOS;XAMARIN_IOS_UNIFIED</DefineConstants>
+		<SupportedOSPlatformVersion>14.2</SupportedOSPlatformVersion>
+		<TargetPlatformMinVersion>14.2</TargetPlatformMinVersion>
+		<TargetPlatformVersion>15.4</TargetPlatformVersion>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="$(TargetFramework.ToLower().StartsWith('net6')) and $(_IsCatalyst)">
+		<SupportedOSPlatformVersion>14.2</SupportedOSPlatformVersion>
+		<TargetPlatformMinVersion>14.2</TargetPlatformMinVersion>
+		<TargetPlatformVersion>15.4</TargetPlatformVersion>
```

Comment:
@jeromelaban I'm not particularly happy that we have to add the TargetPlatformVersion here but it seemed to be the only way I could get this package to build. The MSAL package has SupportedOSPlatformVersion set to 14.2 but ships package with TargetPlatformVersion of 15.4 (from what I can tell). 
If I don't specify TargetPlatformVersion it throws an error stating that TargetPlatformVersion is being set to 12.3.

Reply:
Yes, you have to explicitly specify that, otherwise it chooses the version from the currently installed workloads (as far as I can tell)